hydra-plutus/src/Hydra/Contract/Error.hs

```diff
@@ -0,0 +1,8 @@
+module Hydra.Contract.Error where
+
+import Hydra.Prelude (Show, Text)
+
+-- | Types which are used to describe errors as short error codes in scripts.
+class Show a => ToErrorCode a where
```

Comment:
This is actually not needed in this PR. We should not introduce this constraint right now.